util/updateXML/content/sources/Job/Jobs.py

```diff
@@ -82,15 +82,24 @@ def __init__(self):
         parser.add_option('--update-nrt', action="store_true", dest="update_nrt", help=u"generate output files without gathering them in an output folder to replace")
         options, _ = parser.parse_args()
 
-        if not options.job or not options.origin or not options.version or not options.outputs_path:
+        if not options.job or \
```

Comment:
The first if check if there is 1 or more problems with command arguments. If it's the case we print 1 error message for each problem.
This way we can print every possible problem we have found all at once.

Reply:
I would propose to avoid duplicating the test :
```
error = False
if not options.outputs_path and not options.update_nrt:
  print (...
  error = True
...

if error:
  sys.exit(1)
```